### ISSUE TITLE
Fast Navigation

### ISSUE THREAD

**Issue by Atique29** [sylikc/jpegview#26]
Is it possible to jump to let's say 347th photo from 1?
Is there any key-bind for this?

**Comment by sylikc** [comment]
I don't believe there is currently a keybind for this, though a "goto" feature may be something to be added
